requirements.txt

```diff
@@ -28,9 +28,10 @@ augly==0.1.10
 hyperopt==0.2.3
 gym==0.19.0
 scipy==1.4.1
+protobuf==4.21.4  # For running RL attacks
 
 # Azure packages
-azure-storage-blob==12.14.1
+azure-storage-blob==12.1.0
```

Comment:
Any reason why we needed to downgrade?

Reply:
Other package conflicts. For `azure-storage-blob==12.14.1`, `type-extensions` version to be higher so had to downgrade it.